### PR TITLE
fix(dashboards): Revert the change that causes sort to be wiped

### PR DESCRIPTION
**Pull request by shruthilayaj**
The sort field on table and ton-n widgets were being reset
due to a change introduced in #32208

Will add tests in a follow up so this can be promoted
to prod ASAP

### CODE REVIEW

**Review by wmak — APPROVED** [review]

File: static/app/views/dashboardsV2/widgetBuilder/utils.tsx [hunk 1]
```diff
@@ -92,8 +92,7 @@ export function normalizeQueries(
   }
 
   if ([DisplayType.TABLE, DisplayType.TOP_N].includes(displayType)) {
-    // Reset orderby
-    return queries.map(query => ({...query, orderby: ''}));
+    return queries;
```
Comment (dashed) [inline static/app/views/dashboardsV2/widgetBuilder/utils.tsx:95]: needs tests for checking this 😭

Comment (shruthilayaj) [reply static/app/views/dashboardsV2/widgetBuilder/utils.tsx:95]: @dashed I'm making a separate PR to add test coverage (like literally rn)

Comment (dashed) [reply static/app/views/dashboardsV2/widgetBuilder/utils.tsx:95]: @shruthilayaj thanks 🌮

**Review by dashed — APPROVED** [review]

**Comment by shruthilayaj** [comment]
Visual snapshots were running for 35 minutes...so just decided to push a test here

**Pull request merged**